src/main/kotlin/org/rust/ide/formatter/utils.kt

```diff
@@ -0,0 +1,73 @@
+package org.rust.ide.formatter
+
+import com.intellij.lang.ASTNode
+import com.intellij.psi.PsiElement
+import com.intellij.psi.tree.IElementType
+import com.intellij.psi.tree.TokenSet.orSet
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.RustCompositeElementTypes.*
+import org.rust.lang.core.psi.RustTokenElementTypes.*
+import org.rust.lang.core.psi.impl.RustFile
+import com.intellij.psi.tree.TokenSet.create as ts
+
+internal val KEYWORDS = ts(*IElementType.enumerate { it is RustKeywordTokenType })
```

Comment:
The `internal` modifier is pretty useless for us, because we have only one module. It's OK to make this just public.